module-utils/phonenumber/PhoneNumber.cpp

```diff
@@ -30,15 +30,17 @@ PhoneNumber::PhoneNumber(const std::string &phoneNumber, country::Id defaultCoun
     : countryCode(defaultCountryCode)
 {
     auto &util = *phn_util::GetInstance();
-    util.ParseAndKeepRawInput(phoneNumber, country::getAlpha2Code(countryCode), &pbNumber);
+    auto number = phoneNumber;
+    number.erase(std::remove_if(number.begin(), number.end(), &PhoneNumber::CharacterToRemove), number.end());
```

Comment:
You don't have to use `&` when dealing with pointer to function ;)

Reply:
Done